Cargo.toml

```diff
@@ -3,22 +3,18 @@ name = "cidr-utils"
 version = "0.5.10"
 authors = ["Magic Len <len@magiclen.org>"]
 edition = "2021"
-repository = "https://github.com/magiclen/cidr-utils"
+repository = "https://github.com/magiclen/cidr-utils.git"
 homepage = "https://magiclen.org/cidr-utils"
 keywords = ["cidr", "ip", "ipv4", "ipv6"]
 categories = ["parser-implementations", "data-structures", "network-programming"]
 description = "This crate provides data structures and functions to deal with IPv4 CIDRs and IPv6 CIDRs."
-readme = "README.md"
 license = "MIT"
-include = ["src/**/*", "Cargo.toml", "README.md", "LICENSE"]
+rust-version = "1.58"
 
 [dependencies]
-regex = "1"
-once_cell = "1"
-
```

Comment:
The empty lines are used to group the crates by their relations. I don't want to remove them.

Reply:
For such a short list, it doesn't make much difference. The convention I follow is strict alphabetic ordering, since it interacts nicely with tooling like `cargo add`.